src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -93,6 +105,30 @@ const ImagesTable = ({ selectedBlueprint }: ImageTableProps) => {
     setPage(1);
     setPerPage(perPage);
   };
+  const [buildBlueprint, { isLoading: imageBuildLoading }] =
+    useComposeBlueprintMutation();
+
+  const onBuildHandler = async () => {
+    selectedBlueprint && (await buildBlueprint({ id: selectedBlueprint }));
+  };
+  const [isOpen, setIsOpen] = useState<boolean>(false);
```

Comment:
We don't need to specify `<boolean>` here, Typescript can infer it since the hook was initialized with `false`.